src/generate/CodeModelAzImpl.ts

```diff
@@ -2269,10 +2358,32 @@ export class CodeModelCliImpl implements CodeModelAz {
         return extensions && 'x-ms-examples' in extensions ? extensions['x-ms-examples'] : {};
     }
 
+    public get Example(): unknown {
+        return this.Examples[this.currentExampleIndex];
```

Comment:
will you do a check of currentExampleIndex before returning the value? the default of currentExampleIndex is -1

Reply:
Sure. I will add check logic for all similar function.

Reply:
added.